src/advanced_inv.cpp

```diff
@@ -1966,3 +2020,77 @@ void advanced_inventory::draw_minimap()
         g->u.draw(minimap, g->u.posx(), g->u.posy(), player_invert);
     }
 }
+
+void advanced_inventory::set_vehicle(aim_location sel)
+{
+    // setup on the square itself
+    squares[AIM_VEHICLE].set_vehicle(squares[sel]);
+    // setup for post AIM_VEHICLE selection
+    panes[src].area   = AIM_VEHICLE;
+    panes[src].index  = 0;
+    panes[src].recalc = true;
+    panes[src].redraw = true;
+}
+
+void advanced_inv_area::set_vehicle(advanced_inv_area &s)
+{
+    // this only works for the AIM_VEHICLE square!
+    if(id != AIM_VEHICLE) {
+        return;
+    }
+    if(s.veh == nullptr) {
+        debugmsg("[bug]: squares[sel].veh == nullptr!");
+        return;
```

Comment:
OK, having flunked the sound PR I'm not likely to fix this immediately.  Sorry, DavidKeaton.